File: 0058-length-of-last-word/0058-length-of-last-word.py
class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        cnt = 0;
        for i in range(len(s)-1,-1,-1):
            if(s[i]== ' '):
                if cnt>0 :
                    return cnt
                continue
            cnt+=1;
        return cnt

File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
           String trim = s.trim();
        char[] chars = trim.toCharArray();
        int count = 0;
        for (int i = chars.length - 1; i >= 0; i--) {
            count++;
            if(chars[i]==' '){
                return count-1;
            }
        }
        return count;
    }
}